src/components/earn/PoolCard.tsx

```diff
@@ -110,6 +110,9 @@ export default function PoolCard({ stakingInfo }: { stakingInfo: StakingInfo })
   const valueOfTotalStakedAmountInUSDC =
     valueOfTotalStakedAmountInWETH && USDPrice?.quote(valueOfTotalStakedAmountInWETH)
 
+  // detect if staking is ended
+  const active = stakingInfo?.periodFinish ? stakingInfo.periodFinish.getMilliseconds() > Date.now() : true
```

Comment:
Date.now() is machine dependent, can we use useCurrentBlockTimestamp()? better yet, add this to the staking info so we don't fetch in every pool card